src/applications/vaos/new-appointment/redux/reducer.js

```diff
@@ -837,6 +837,11 @@ export default function formReducer(state = initialState, action) {
           system => `${system.address?.city}, ${system.address?.state}`,
         );
         initialSchema.required = ['communityCareSystemId'];
+      } else {
```

Comment:
Still need to unset the question, if the flag is on and there's more than one site